src/pages/ExamsPage/components/ExamSelection.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  MenuItem, SearchField,
+  SelectMenu,
+} from '@edx/paragon';
+import PropTypes from 'prop-types';
+
+const ExamSelection = ({ exams, onSelect }) => {
+  const { formatMessage } = useIntl();
+  const [searchText, setSearchText] = useState('');
+
+  const placeholderMessage = formatMessage({
```

Comment:
could these messages be moved to a message specific file? I typically haven't seen them defined within the components.

Reply:
Also are we considering internationalization?

Reply:
oh yeah you're right, this should be done using a separate file.

Reply:
ok this should be fixed. Turns out I had already set that file up I just didn't make use of it.